pkg/generator/pcf-artifacts.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generator
+
+import (
+	"bytes"
+	"log"
+	"text/template"
+
+	"github.com/GoogleCloudPlatform/gcp-service-broker/utils"
+)
+
+const (
+	appName         = "gcp-service-broker"
+	appDescription  = "A service broker for Google Cloud Platform services."
+	stemcellOs      = "ubuntu-trusty"
+	stemcellVersion = "3586.42"
+
+	buildpack     = "go_buildpack"
+	goPackageName = "github.com/GoogleCloudPlatform/gcp-service-broker"
+	goVersion     = "go1.10"
+
+	copyrightHeader = `# Copyright the Service Broker Project Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is AUTOGENERATED by ./gcp-service-broker generate, DO NOT EDIT IT.
+
+---`
+	manifestYmlTemplate = copyrightHeader + `
+applications:
+  - name: {{.appName}}
+    product_version: "{{.appVersion}}"
+    metadata_version: "1.0"
+    label: 'GCP Service Broker'
+    description: '{{.appDescription}}'
+    memory: 1G
```

Comment:
Any reason why these are not const values as well?
Even though they won't be changed often, if ever, still might be a good idea for consistency.

Reply:
I originally tried to just pull out the common vars, but I like your way better.